<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KanbanBoard</class>
 <widget class="QWidget" name="KanbanBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>836</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1360</width>
    <height>836</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1360</width>
    <height>836</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fff;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="vert">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>68</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>380</width>
         <height>68</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>To Do</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="verticalWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>380</width>
         <height>9999</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="todoLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="addNewButton">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>30</pointsize>
            <bold>true</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #fff; color: #6096BA</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vert_2">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>68</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>380</width>
         <height>68</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>In Progress</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="verticalWidget_2" native="true">
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>380</width>
         <height>9999</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="inprogressLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vert_3">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>68</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>380</width>
         <height>68</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Completed</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="verticalWidget_3" native="true">
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>380</width>
         <height>9999</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="completedLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
